basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,237 @@
 const helpers = require('./project-1');
 
 // start testing!
+
+describe('testing funtions in project-1.js', () => {
+
+    it('test multiplyby10 function', () => {
+        // arrange
+        const exp = 2 * 10
```

Comment:
What about the situation where You have a `0`? Or what about the situation where You provide string? Testing is all about checking all the different cases.